### ISSUE TITLE
Ping (Feature Change Request)

### ISSUE THREAD

**Issue by malicioustoker** [wireapp/wire-ios#1580]
Currently, if someone pings you while they have timed messages on, the notification just states that someone sent you a message. Seeing that a ping really isn’t related to privacy, would it be possible to change the notification when receiving a ping from someone with times messages on to just state that “X has pinged you” or something like that?

**Comment by mikeger** [comment]
Hey @malicioustoker ! Thanks for the feature request. We are going to pass it to design to get the verification. @Yorgos-V

**Comment by 0ki** [comment]
I would object to having "X" (the username) in any notifications when timed messages are on or message previews are off.

**Comment by malicioustoker** [comment]
I completely agree with you on that, but even for a ping? I feel that a ping could have the username. Defiantly not messages though 

— Reverend Glen

**Comment by malicioustoker** [comment]
I completely agree with you on that, but even for a ping? I feel like a ping would be fine to have a username appear saying [Username] pinged you, or whatever it says

**Comment by 0ki** [comment]
If you display a username for a ping then you are basically displaying both the username and the content of the communication. (Not that I would find displaying just one a good idea even.)

Many people use their real names on Wire, which would mean anyone seeing that notification on the lockscreen could be certain that that person is in your contact list.

**Comment by malicioustoker** [comment]
🤔 okay, how about a compromise and giving the power back to the user, allowing them to choose if a a ping doesn’t follow the same privacy as an actual message? 🤷🏼‍♂️
